### ISSUE TITLE
Tasks provided by plugin should declare group and description property values

### ISSUE THREAD

**Issue by bmuschko** [gogradle/gogradle#117]
Right now it's hard for a user to explore tasks provided by the plugin. Executing `gradlew tasks --all` renders a list of tasks but it's somewhat unclear what the tasks belong to and what their purpose is.

```
Other tasks
-----------
build
check
clean
cLion
coverage
dependencies
fmt
gogland
goIdea
init
installDependencies
lock
phpStorm
prepare
pyCharm
resolveBuildDependencies
resolveTestDependencies
rubyMine
showGopathGoroot
test
vendor
vet
vscode
webStorm
```

I propose you use the group `Gogradle` or `Go development` and a descriptive name for all tasks introduced by the plugin.

**Comment by blindpirate** [comment]
Sure, I haven't realized this issue before, thank you.

**Comment by blindpirate** [comment]
Fixed by `0.6.5`.
